backend/models/invitations.js

```diff
@@ -121,7 +121,7 @@ invitations.create = async (email, teamspace, job, username, permissions = {}) =
 		throw responseCodes.INVALID_MODEL_ID;
 	}
 
-	if (await teamspaceObject.hasReachedLicenceLimit()) {
+	if (await User.hasReachedLicenceLimit(teamspaceObject)) {
```

Comment:
This should just take the teamspace name and leave User to fetch the teamspace object?

Reply:
the problem with that is that you are fetching several times the same user when you can do it just onces and pass it through to check whatever

Reply:
apart from that's not really the use case here, once again we're fetching the user object just to call this function.

Reply:
Can we do something similar to the `isMemberOfTeamspace ` with `hasReachedLicenceLimit`? Once again it's getting the object just to call this function here, and the only other place that his is used is within `user.js`